website/docs/docs/cloud-integrations/set-up-snowflake-native-app.md

```diff
@@ -28,7 +28,7 @@ The following are the prerequisites for dbt Cloud and Snowflake.
 
 - You must have a dbt Cloud account on the Enterprise plan that's in an AWS Region or Azure region. If you don't already have one, please [contact us](mailto:sales_snowflake_marketplace@dbtlabs.com) to get started.
     - Currently, Semantic Layer is unavailable for Azure ST instances and the **Ask dbt** chatbot will not function in the dbt Snowflake Native App without it. 
-- Your dbt Cloud account must have permission to create a [service token](/docs/dbt-cloud-apis/service-tokens). For details, refer to [Enterprise permissions](/docs/cloud/manage-access/enterprise-permissions).
+- Your dbt Cloud account must have permission to create a [service token](/docs/dbt-cloud-apis/service-tokens) assigned the `Manage marketplace apps` role. For details, refer to [Enterprise permissions](/docs/cloud/manage-access/enterprise-permissions).
```

Comment:
Lower down on this page, we **also** say to grant Job Admin, Metadata, Semantic Layer Only. IS there a reason we're just mentioning this one or should we mention these as well?

https://github.com/dbt-labs/docs.getdbt.com/pull/5815/files#diff-770944d95df777c9ab980c64ab039e22058f83e3eaa322e78fca7429f397c128L71-L73

Reply:
That's a good catch. That shouldn't be in that spot, reverting it to the way it was and relying on the below info only!